packages/generator-langium/templates/core/src/language/language-id-module.ts

```diff
@@ -23,7 +25,7 @@ export type <%= LanguageName %>Services = LangiumServices & <%= LanguageName %>A
  * declared custom services. The Langium defaults can be partially specified to override only
  * selected services, while the custom services must be fully specified.
  */
-export const <%= LanguageName %>Module: Module<<%= LanguageName %>Services, PartialLangiumServices & <%= LanguageName %>AddedServices> = {
+export const <%= LanguageName %>Module: Module<<%= LanguageName %>Services, PartialLangiumCoreServices & PartialLangiumLSPServices & <%= LanguageName %>AddedServices> = {
```

Comment:
I would prefer to revert this change and introduce `PartialLangiumServices` again.

Reply:
Seems like this is the direction overall, and having `PartialLangiumServices` sourced from the lsp export instead.

Reply:
fixed.